apps/01PigLatin.js

```diff
@@ -50,42 +50,42 @@ function pigLatin(word) {
 
 
   if (vowelIndex === 0) {
-      return word + 'yay';
-    } else {
-      return restWord + firstPart + 'ay';
-    }
+    return word + 'yay';
+  } else {
+    return restWord + firstPart + 'ay';
+  }
 }
 
 
 function getPrompt() {
   rl.question('word ', (answer) => {
-      console.log(pigLatin(answer));
-      getPrompt();
-    });
+    console.log(pigLatin(answer));
+    getPrompt();
+  });
 }
 
 // Tests
 
 if (typeof describe === 'function') {
 
   describe('#pigLatin()', function() {
-      it('should translate a simple word', function() {
-          assert.equal(pigLatin('car'), 'arcay');
-          assert.equal(pigLatin('dog'), 'ogday');
-        });
-      it('should translate a complex word', function() {
-          assert.equal(pigLatin('create'), 'eatecray');
-          assert.equal(pigLatin('valley'), 'alleyvay');
-        });
-      it('should attach "yay" if word begins with vowel', function() {
-          assert.equal(pigLatin('egg'), 'eggyay');
-          assert.equal(pigLatin('emission'), 'emissionyay');
-        });
-      it('should auto lowercase word before translation', function() {
-          assert.equal(pigLatin('HeLlO'), 'ellohay');
-          assert.equal(pigLatin('RoCkEt'), 'ocketray');
-        });
+    it('should translate a simple word', function() {
+      assert.equal(pigLatin('car'), 'arcay');
+      assert.equal(pigLatin('dog'), 'ogday');
+    });
+    it('should translate a complex word', function() {
+      assert.equal(pigLatin('create'), 'eatecray');
+      assert.equal(pigLatin('valley'), 'alleyvay');
+    });
+    it('should attach "yay" if word begins with vowel', function() {
+      assert.equal(pigLatin('egg'), 'eggyay');
+      assert.equal(pigLatin('emission'), 'emissionyay');
+    });
+    it('should auto lowercase word before translation', function() {
+      assert.equal(pigLatin('HeLlO'), 'ellohay');
+      assert.equal(pigLatin('RoCkEt'), 'ocketray');
     });
+  });
 } else {
 
   getPrompt();
```

Comment:
I'm wondering if we should have put something in the getPrompt parenthesis like "Give me a word" to actually prompt someone to give us a word... or would this be the wrong place to put it? Still a little confused on this.